rootston/desktop.c

```diff
@@ -93,6 +93,36 @@ static struct wlr_subsurface *subsurface_at(struct wlr_surface *surface,
 	return NULL;
 }
 
+static struct wlr_xdg_surface_v6 *xdg_v6_popup_at(
```

Comment:
`xdg_v6_popup_at()` was done by intuition and trial and error. Maybe someone smarter than me can simplify this.